### ISSUE TITLE
A ConvNet for the 2020s

### ISSUE THREAD

**Issue by Arata2887** [Arata2887/issue_copy#4]
1.どんな論文? /What does this work do?

- https://arxiv.org/abs/2201.03545
- [ConvNeXt.pptx](https://github.com/jr-tagawalab-nttcom/paper-survey/files/11485322/ConvNeXt.pptx)

- Propose ConvNeXt, a pure ConvNet module, which compete favorably with Transformers in terms of accuracy, scalability and robustness across all major benchmarks.
- Bridge the gap between the pre-ViT and post-ViT eras for ConvNets, and test the limits of what a pure ConvNet can achieve by attempting a series of design decisions.

2.先行研究と比べてどこがすごい? /What makes this work greater than existing works?

By incorporating many Transformer architectural choices into ConvNet, ConvNeXt can compete favorably with state-of-the-art hierarchical vision Transformers across multiple computer vision benchmarks, while retaining the simplicity and efficiency of standard ConvNets.

![Figure 1](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/0b02a07f-f44b-46a9-a8fd-87afa816bd3f)

Figure 1 shows that ImageNet-1K（ImageNet-1K Trained and ImageNet-22K pre-trained） classification results for ConvNets(purple) and vision Transformers(orange). Each bubble’s area is proportional to FLOPs of a variant in a model family. ImageNet-1K/22K models here take 224x224/384x384 images respectively.

![Table 12](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/56d102ca-8b41-4f3b-a3c9-6fa753d0b461)

Table 12 shows the Inference throughput comparisons of **Swin Transformer** and **ConvNeXt** on A100 GPU. ConvNeXts’ advantage is significantly greater, sometimes up to 49% faster

3.技術や手法のキモはどこ? /What is the heart of this technology or method?

In some ways, **Observations of the paper are surprising while ConvNeXt is not completely new—many design choices have all been examined separately over the last decade**, but not collectively.
Starting point of ConvNeXt is a ResNet-50 model. 
Authors first train it with similar training techniques used to train vision Transformers and obtain much improved results compared to the original ResNet-50. 
Then, Authors attempting a series of design decisions, which Authors summarized as **macro design**, **ResNeXt**, **inverted bottleneck**, **large kernel size**, and various layer-wise **micro designs**. 

![Figure 2](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/a84d3cfe-3c4c-4f1e-b6f5-f65fefb041f9)

Figure 2 show the procedure and the results Authors are able to achieve with each step of the “network modernization”(All models are trained and evaluated on ImageNet-1K).

4.どうやって有効だと検証した? /How is this work validated?

![Table 1 (upper)](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/3ec97036-a3bd-4933-a86d-2f2da1414f90)

Table 1_upper(V100 base) shows the result comparison with two recent Transformer variants, DeiT and Swin Transformers, as well as two ConvNets from architecture search - RegNets, EfficientNets and EfficientNetsV2. ConvNeXt competes favorably with two strong ConvNet baselines RegNet and EfficientNet) in terms of the accuracy-computation trade-off, as well as the inference throughputs.

![Table 1 (lower)](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/af6c7cf3-034c-4784-a205-27a9a28de3d3)

Table 1_lower(V100 base) shows the present results with models fine-tuned from ImageNet-22K pre-training.

![Table 2](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/d883382d-7bf8-4285-b054-938faa2aa9e3)

Results for ImageNet-1K at 224x224 resolution are in Table 2. In this ablation, Authors examine if our ConvNeXt block design is generalizable to ViT-style isotropic architectures which have no downsampling layers and keep the same feature resolutions (e.g. 14x14) at all depths. Authors construct isotropic ConvNeXt-S/B/L using the same feature dimensions as ViT-S/B/L (384/768/1024). Depths are set at 18/18/36 to match the number of parameters and FLOPs.

5.議論はある? /Any discussion for this work?

6.読んでいてわからなかったところは? /What don't you understand for this paper?

7.公開コードやデータセットは? /Public codes or datasets?

https://github.com/facebookresearch/ConvNeXt

8.次に読むべき論文は? /What should we read next?
